File: app/src/main/java/com/example/dronedeliveryservice/OrderInfo.java
package com.example.dronedeliveryservice;


import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.ImageButton;
import android.widget.TextView;import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import androidx.appcompat.app.AppCompatActivity;
import android.view.View.OnClickListener;
import com.google.android.gms.maps.CameraUpdateFactory;
import com.google.android.gms.maps.GoogleMap;
import com.google.android.gms.maps.MapFragment;
import com.google.android.gms.maps.OnMapReadyCallback;
import com.google.android.gms.maps.SupportMapFragment;
import com.google.android.gms.maps.model.BitmapDescriptorFactory;
import com.google.android.gms.maps.model.LatLng;
import com.google.android.gms.maps.model.Marker;
import com.google.android.gms.maps.model.MarkerOptions;


import androidx.appcompat.app.AppCompatActivity;
public class OrderInfo extends AppCompatActivity implements OnMapReadyCallback,View.OnClickListener {


    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_order_info);

        Button drone_info_btn1 = (Button) findViewById(R.id.drone_info_btn);
        drone_info_btn1.setOnClickListener(this);

        SupportMapFragment mapFragment = (SupportMapFragment) getSupportFragmentManager()
                .findFragmentById(R.id.map);
        mapFragment.getMapAsync(this);


    }


            @Override
            public void onClick(View v) {
                switch(v.getId()) {

                    case R.id.drone_info_btn:
                      Intent drone_info_start = new Intent(getApplicationContext(), Bay1.class);
                      startActivity(drone_info_start);
                      break;



            }
    }


    @Override
    public void onMapReady(GoogleMap googleMap) {
        GoogleMap map_satellite = googleMap;
        map_satellite.setMapType(map_satellite.MAP_TYPE_SATELLITE);
        LatLng lexington = new LatLng(37.789426336756414, -79.43785182364378);
        map_satellite.animateCamera(CameraUpdateFactory.newLatLngZoom(lexington, 15f));
        googleMap.addMarker(new MarkerOptions()
                .position(lexington)
                .title("Marker at Mallory"))
                .setIcon(BitmapDescriptorFactory.fromResource(R.drawable.airplane_marker));
    }
}

File: app/src/main/java/com/example/dronedeliveryservice/OrderInfo.kt
package com.example.dronedeliveryservice

import android.content.Intent
import android.os.Bundle
import android.widget.ArrayAdapter
import android.widget.Spinner
import androidx.appcompat.app.AppCompatActivity
import kotlinx.android.synthetic.main.activity_order_info.*

class OrderInfo : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_order_info)

        dashboard_btn1.setOnClickListener{
            val intent1 = Intent(this, BayDashboard::class.java)
            startActivity(intent1)
        }

        droneinfo_btn.setOnClickListener{
            val intent1 = Intent(this, Bay1::class.java)
            startActivity(intent1)
        }

        //Dimensions Drop Down Menu

        val spinner: Spinner = findViewById(R.id.dimensions_spinner)
// Create an ArrayAdapter using the string array and a default spinner layout
        ArrayAdapter.createFromResource(
                this,
                R.array.dimensions,
                android.R.layout.simple_spinner_item
        ).also { adapter ->
            // Specify the layout to use when the list of choices appears
            adapter.setDropDownViewResource(android.R.layout.simple_spinner_dropdown_item)
            // Apply the adapter to the spinner
            spinner.adapter = adapter
        }

//Weight Class Drop Down Menu

        val spinner1: Spinner = findViewById(R.id.weight_class_spinner)
// Create an ArrayAdapter using the string array and a default spinner layout
        ArrayAdapter.createFromResource(
            this,
            R.array.weight_class,
            android.R.layout.simple_spinner_item
        ).also { adapter ->
            // Specify the layout to use when the list of choices appears
            adapter.setDropDownViewResource(android.R.layout.simple_spinner_dropdown_item)
            // Apply the adapter to the spinner
            spinner1.adapter = adapter
        }

        dashboard_nav_button.setOnClickListener {
            val intent = Intent(this, BayDashboard::class.java)
            startActivity(intent)
        }

    }




}
